controllers/machine_controller_test.go

```diff
@@ -40,6 +41,107 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var _ = Describe("Machine Reconciler", func() {
```

Comment:
We are trying to avoid using ginkgo tests going forward but for this particular case:
To use testenv with go tests in ./controllers, we need to make changes in `suite_test.go` similar to `controlplane/kubeadm/internal/suite_test.go`. 
Currently all tests that use testenv in ./controllers package are ginkgo tests, so I think we will need to change them all together with the `suite_test.go`.